collectors/mesos/agent/state.go

```diff
@@ -50,22 +50,22 @@ type frameworkInfo struct {
 }
 
 type executorInfo struct {
-	ID        string           `json:"id"`
-	Name      string           `json:"name"`
-	Container string           `json:"container"`
-	Labels    []executorLabels `json:"labels,omitempty"` // labels are optional
-	Tasks     []taskInfo       `json:"tasks,omitempty"`
+	ID        string       `json:"id"`
+	Name      string       `json:"name"`
+	Container string       `json:"container"`
+	Labels    []stateLabel `json:"labels,omitempty"` // labels are optional
+	Tasks     []taskInfo   `json:"tasks,omitempty"`
 }
 
-type executorLabels struct {
+type stateLabel struct {
```

Comment:
I actually did this at first, but then backed it out and went with `stateLabel` because I thought `label` was too vague. I'll switch it back if you're happy with `label`.